### ISSUE TITLE
Detect and fix direct imports of sap/m/URLHelper

### ISSUE THREAD

**Issue by flovogt** [UI5/linter#1138]
The UI5 linter should report an error when `sap/m/URLHelper` is imported directly as a module. URLHelper is not intended to be consumed via a standalone module import but must be accessed through sap/m/library.

# Wrong usage
```js
sap.ui.define([
  "sap/m/URLHelper"
], (URLHelper) => {
  "use strict";
  onInit() {
    URLHelper.redirect("https://www.openui5.org", true);
  }
});
```

# Recommended usage

```js
sap.ui.define([
  "sap/m/library"
], (mLibrary) => {
  "use strict";
  onInit() {
    mLibrary.URLHelper.redirect("https://www.openui5.org", true);
  }
});
```

**Comment by flovogt** [comment]
- Detection is part of backlog item CPOUI5FOUNDATION-1369
- Autofix is part of backlog item CPOUI5FOUNDATION-1370
